test/units/end.sh

```diff
@@ -9,5 +9,9 @@ set -o pipefail
 # Here, the redundant '[ ]' in the pattern is required in order not to match the logged command itself.
 (! journalctl -q -o short-monotonic --grep 'Warning: cannot close sd-bus connection[ ].*after fork' >>/failed)
 
+# Check if sd-executor doesn't complain about not being able to (de)serialize stuff
+(! journalctl -q -o short-monotonic --grep "[F]ailed to parse serialized line" >>/failed)
```

Comment:
What is the reason for the `[F]` construct?

Reply:
The test scripts run with `set -x` and are eventually logged into the journal. The `[F]` stuff is necessary so the grep doesn't match the executed command itself.